### ISSUE TITLE
Windows builds running on Windows 7 should check for KB2533623 and warn if it is not installed

### ISSUE THREAD

**Issue by dblohm7** [tailscale/tailscale#6647]
This is our most frequent logged panic on Windows.

On Windows 7, Wintun requires [KB2533623](https://support.microsoft.com/en-us/topic/microsoft-security-advisory-insecure-library-loading-could-allow-remote-code-execution-486ea436-2d47-27e5-6cb9-26ab7230c704) to be installed, which might not have been done if Windows 7 users have not kept up to date (which, given that they're already on a EOL OS, is very possible).

It would be nice if we could explicitly query Windows Update to check for that, but if not, calling `LoadLibraryExW` with the `LOAD_LIBRARY_SEARCH_APPLICATION_DIR` flag will return `ERROR_INVALID_PARAMETER` if the update is not installed. If we could detect that and at least throw up a notification of some kind, that would help immensely.

**Comment by bradfitz** [comment]
What's the panic?

**Comment by dblohm7** [comment]
```
07:44:47 | golang.zx2c4.com/wintun.(*lazyProc).Addr(...)
07:44:47 | golang.zx2c4.com/wintun@v0.0.0-20211104114900-415007cec224/dll.go:60
07:44:47 | golang.zx2c4.com/wintun.CreateAdapter({0x14071f101?, 0x13ff69411?}, {0x14071f101?, 0x9?}, 0xc000041430?)
07:44:47 | golang.zx2c4.com/wintun@v0.0.0-20211104114900-415007cec224/wintun.go:92 +0x19f
07:44:47 | golang.zx2c4.com/wireguard/tun.CreateTUNWithRequestedGUID({0x14071f101, 0x9}, 0xc000004940?, 0x500)
07:44:47 | golang.zx2c4.com/wireguard@v0.0.0-20220703234212-c31a7b1ab478/tun/tun_windows.go:73 +0x46
07:44:47 | golang.zx2c4.com/wireguard/tun.CreateTUN(...)
07:44:47 | golang.zx2c4.com/wireguard@v0.0.0-20220703234212-c31a7b1ab478/tun/tun_windows.go:65
07:44:47 | tailscale.com/net/tstun.New(0x1407754d0, {0x14071f101?, 0x140c53940?})
07:44:47 | tailscale.com@v1.30.2/net/tstun/tun.go:55 +0x87
07:44:47 | main.startIPNServer.func1()
07:44:47 | tailscale.com@v1.30.2/cmd/tailscaled/tailscaled_windows.go:268 +0x5e
07:44:47 | main.startIPNServer.func2()
07:44:47 | tailscale.com@v1.30.2/cmd/tailscaled/tailscaled_windows.go:323 +0x103
07:44:47 | created by main.startIPNServer
07:44:47 | tailscale.com@v1.30.2/cmd/tailscaled/tailscaled_windows.go:318 +0x1d5
```

**Comment by DentonGentry** [comment]
Fixed in [1.36.1](https://github.com/tailscale/tailscale/releases/tag/v1.36.1) by updating wintun.
